test/mocha.opts

```diff
@@ -2,4 +2,3 @@
 --require chai/register-expect
 --require source-map-support/register
 --timeout 10000
---throw-deprecation
```

Comment:
This prevents a crash on node 16, chai makes use of a deprecated "exports" field in its package.json